test/other/benchmarkMemPoolTypes.cpp

```diff
@@ -7,6 +7,16 @@
 #include <cstddef>
 #include <vector>
 
+namespace {
+    namespace objsizes {
+        // common structs used in the engine
+        constexpr size_t micro=64; // CMatrix44f
+        constexpr size_t small=128; // SolidObjectGroundDecal
+        constexpr size_t medium=752; // PlasmaRepulser
+        constexpr size_t large=1472; // CFeature
+    }
```

Comment:
Is PlasmaRepulser actually common? What about CUnit, perhaps CWeapon, script?, CEG particles?.

Reply:
> are you suggesting to add a test/benchmark with randomized object size?

I was about to suggest (I don't know the exact size):
```diff
-constexpr size_t medium = 752; // PlasmaRepulser
+constexpr size_t medium = NNN; // CUnit, one of the most important and commonly used structs
```
but after your explanation now I think I am just suggesting
```cpp
/* Note, these aren't the most commonly used structs (like CUnit),
 * but they are used in the benchmark because they are
 * the largest in their category (page size in allocator). */
 ```

Reply:
ok. done